### ISSUE TITLE
Oxygen Builder 4.0, new size and spacing style, not a fan.

### ISSUE THREAD

**Issue by madjedo** [soflyy/oxygen-bugs-and-features#2625]
### Not a fan of the new styling and ux for the size and spacing.

![size_and_spacing_new_ui](https://user-images.githubusercontent.com/25926337/147598714-2ef0a6da-4feb-4492-9b9f-990fdcdcf40f.png)

I liked it the way it was before. I think most people already grasp the concept of margins (outside) and padding (inside). The new visual representation feels "clunky" with the margin boxes being so much spaced out from each other and the "apply to all" being so far on bottom.

**Do you agree and would like to see something else or revert back to normal? Vote this up and comment if you have a suggestion for better UX regarding styling size and spacing.**

My suggestions would be to add a question mark right next to the 'Size & Spacing' text, and on hoover explain the concept, like a tooltip. Could be something like this:

![suggestion2-question-mark-hover_size_and_spacing_new_ui](https://user-images.githubusercontent.com/25926337/147600767-14b12466-a0b0-4004-9e3e-a5c6fb853a8e.png)

And on hover/click question mark get something inspired by this:

![61f271bb01d3b693943cae6ca63ec311](https://user-images.githubusercontent.com/25926337/147600797-a109a5ab-cfac-4b28-a3ed-13420006ddaf.png)

**Comment by Daniel-Florian** [comment]
I agree - I personally hate the new margins/padding input setup. I see what they were going for but it just looks really odd to me

**Comment by Oskar993** [comment]
Anyone having issue with negative margin?

**Comment by Spellhammer** [comment]
Thank you for your feedback.

**Comment by JTabdesign** [comment]
Agreed, if it ain't broke don't fix it

**Comment by shoelaced** [comment]
Agree it feels unnecessarily visual and less efficient - also makes it difficult to use tab to move from field to field.

**Comment by switchupcb** [comment]
![image](https://user-images.githubusercontent.com/81384235/150030141-5fcb720c-4977-494c-a7dc-aa0f711ad057.png)

**Comment by Zeb4Web** [comment]
Thank you Madjedo, I was opening an issue with the very same matter.
99% of persons using Oxygen know perfectly what it is padding and margin (the other 1% could search) 
We do not need the visual representation of margin and padding.
My priority save space and therefore mouse movements and time.

**Comment by madjedo** [comment]
Another suggestion, this one a bit more developer friendly, so you can have enough space to enter in css variables:

![OxygenUIUX](https://user-images.githubusercontent.com/25926337/152370216-71cf62a4-49a2-4445-a264-11b0b006a375.png)

This one in a 2-column layout. But seeing there is still alot of vertical space left, you could even do this in 1-column layout for even more space for variables and what have you.

You can pay me after you redesign ;)

**Comment by Zeb4Web** [comment]
Madjedo we should meet!
That is the second suggestion you steal… I mean, you rise a moment before me.
The point is in the past we have write a number inside the margin/padding usually 2 (max 3) digits.
Nowadays we insert pieces of CSS, sometimes pretty long, such as: nth-chid or clamp.

There should be a way to see all the value at once, especially for debugging.

You can pay me, as well, after the redesign!
